java/fundamentals/intro-to-java/what-is-java.md

```diff
@@ -22,24 +19,32 @@ inAlgoPool: false
 ---
 ## Content
 
-**Java** is a programming language that follows the *Object-oriented programming paradigm* (also  known as OOP).
-
-One of the particularities of the language that makes it so widely used is that compiled Java code runs on any machine that supports Java, without recompilation. This principle is called *"write once, run anywhere"* (WORA).
+**Java** is a programming language that follows the *Object-oriented programming paradigm*[1] (also known as OOP).
 
-Java classes are compiled to `bytecode`, which any Java Virtual Machine (*JVM*) can understand.
+It was first released in 1995 by Sun Microsystems, and has since grown to be one of the most used programming languages.
 
-In short, Java is trying to be an object-oriented, secure, portable, threaded and dynamic language. We will discuss all these terms in future insights.
+The fact that it's still one of the most-used languages over 25 years later shows how durable it is.
 
-*Note*: You should get comfortable with the acronyms as they will come up regularly in the course of the workouts.
+Java is an **object-oriented, secure, and portable language**. We'll discuss what this means very soon!
 
 ---
 ## Practice
 
-What type of programming language is **Java** considered?
+What type of programming language is **Java** considered to be?
 
 ???
 
 * object-oriented
 * functional
 * quantum
 * esoteric
+
+---
+## Footnotes
+
+[1:Object-oriented programming]
+All you need to know for now is that **object-oriented programming** (OOP) is a *programming pattern* that works with real-life entities. These entities are called **objects**, and they have *attributes* and *behaviors*. 
+
+Let's use the common car analogy to better understand OOP. 
+
+Think of the car as the object. The model, color or age of the car are the object's *attributes*. Whether the engine is on, or whether it's moving are *behaviors* and would exist as functions.
```

Comment:
This was an awesome analogy

Reply:
we have @emmbow to thank for this :muscle: . she came up with it for python, and I just repurposed it.